### ISSUE TITLE
Update copyright.py

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3504]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#3504|
|Type|task|
|Created|2008-10-27 00:26:14Z|

[twisted/copyright.py](twisted/copyright.py) still references 2001-2006, let's update that to 2001-2008.

Attachments:

* [copyright.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/821/82128953f6ed3ae729dd80bb3b748e4bd34b8816/54d56589249d32caaef87fb9ee3dc9ac5f374462.patch) (609 bytes) - added by thijs on 2008-10-27 00:27:33Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__3504 3504
type__task task
reporter__thijs thijs
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1225067174000000 1225067174000000
changetime__1225135795000000 1225135795000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

Great, please apply.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [25138]) Apply copyright.patch: Update copyright information for Twisted.

Author: thijs
Reviewer: exarkun
Fixes: [#3504](https://github.com/twisted/twisted/issues/3504)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
